pex/pip/installation.py

```diff
@@ -99,8 +99,11 @@ def bootstrap_pip():
         # type: () -> Iterator[str]
 
         chroot = safe_mkdtemp()
-        venv = Virtualenv.create(venv_dir=os.path.join(chroot, "pip"), interpreter=interpreter)
-        venv.install_pip(upgrade=True)
+        venv = Virtualenv.create(
```

Comment:
Without this change, the [`test_bdist_pex_under_tox`](https://github.com/pex-tool/pex/pull/2454/files#diff-bf53c6a46383b9feddf4b18293882ce85c33b204fd9ddff47343461cdda054feR80) test fails for Python 3.12 and 3.13 just like the OP case in https://github.com/pex-tool/pex/issues/2413#issue-2312987117:
```
:; tox -epy312-integration -- --devpi -vvsk test_dist_pex_under_tox -n0
...
tests/integration/test_issue_2413.py::test_dist_pex_under_tox bundle: recreate env because python changed version_info=[3, 11, 9, 'final', 0]->[3, 12, 4, 'final', 0] | executable='/usr/bin/python3.11'->'/usr/bin/python3.12'
bundle: remove tox env folder /home/jsirois/dev/pex-tool/pex/.tox/bundle
bundle: install_deps> python -I -m pip install setuptools /home/jsirois/dev/pex-tool/pex
.pkg: recreate env because python changed version_info=[3, 11, 9, 'final', 0]->[3, 12, 4, 'final', 0] | executable='/usr/bin/python3.11'->'/usr/bin/python3.12'
.pkg: remove tox env folder /home/jsirois/dev/pex-tool/pex/.tox/.pkg
.pkg: install_requires> python -I -m pip install 'setuptools>=40.8.0' wheel
.pkg: _optional_hooks> python /tmp/pytest-of-jsirois/pytest-46/test_dist_pex_under_tox0/venv/lib/python3.12/site-packages/pyproject_api/_backend.py True setuptools.build_meta __legacy__
.pkg: get_requires_for_build_sdist> python /tmp/pytest-of-jsirois/pytest-46/test_dist_pex_under_tox0/venv/lib/python3.12/site-packages/pyproject_api/_backend.py True setuptools.build_meta __legacy__
.pkg: get_requires_for_build_wheel> python /tmp/pytest-of-jsirois/pytest-46/test_dist_pex_under_tox0/venv/lib/python3.12/site-packages/pyproject_api/_backend.py True setuptools.build_meta __legacy__
.pkg: prepare_metadata_for_build_wheel> python /tmp/pytest-of-jsirois/pytest-46/test_dist_pex_under_tox0/venv/lib/python3.12/site-packages/pyproject_api/_backend.py True setuptools.build_meta __legacy__
.pkg: build_sdist> python /tmp/pytest-of-jsirois/pytest-46/test_dist_pex_under_tox0/venv/lib/python3.12/site-packages/pyproject_api/_backend.py True setuptools.build_meta __legacy__
bundle: install_package_deps> python -I -m pip install cowsay==5.0
bundle: install_package> python -I -m pip install --force-reinstall --no-deps /home/jsirois/dev/pex-tool/pex/.tox/.tmp/package/29/repro-0.0.1.tar.gz
bundle: commands[0]> /home/jsirois/dev/pex-tool/pex/.tox/bundle/bin/python setup.py bdist_pex --bdist-dir=dist --pex-args=--disable-cache --bdist-all
running bdist_pex
Writing entry to dist/entry
Failed to create pex via /home/jsirois/dev/pex-tool/pex/.tox/bundle/bin/python -s -m pex /tmp/pytest-of-jsirois/pytest-46/test_dist_pex_under_tox0/project --disable-cache --output-file dist/entry --script entry:
Failed to spawn a job for /home/jsirois/dev/pex-tool/pex/.tox/bundle/bin/python: received exit code 1 during execution of `['/tmp/tmpnaa5pd6b/pip/bin/python3.12', '-s', '-E', '-m', 'pip', 'install', '-U', 'pip']` while trying to execute `['/tmp/tmpnaa5pd6b/pip/bin/python3.12', '-s', '-E', '-m', 'pip', 'install', '-U', 'pip']`

bundle: exit 1 (1.88 seconds) /tmp/pytest-of-jsirois/pytest-46/test_dist_pex_under_tox0/project> /home/jsirois/dev/pex-tool/pex/.tox/bundle/bin/python setup.py bdist_pex --bdist-dir=dist --pex-args=--disable-cache --bdist-all pid=508461
  bundle: FAIL code 1 (10.30=setup[8.43]+cmd[1.88] seconds)
  evaluation failed :( (10.33 seconds)
FAILED
```